### ISSUE TITLE
[Bug Report] Tooltip tabindex always set to 0

### ISSUE THREAD

**Issue by lochstar** [ElemeFE/element#11519]
### Element UI version
2.4.0

### OS/Browsers version
macOS / Chrome 66

### Vue version
2.5.16

### Reproduction Link
[https://jsfiddle.net/g9c3oh1d/](https://jsfiddle.net/g9c3oh1d/)

### Steps to reproduce
Focus "Input 1".
Press "TAB" key.

### What is Expected?
"Input 2" should be focused.

### What is actually happening?
The tooltip is being focused.

**Comment by atilkan** [comment]
Your example doesn't work. There is no **label** slot for **el-input**
**I updated it. here** -> https://jsfiddle.net/g9c3oh1d/2/
**Docs**. -> http://element.eleme.io/#/en-US/component/input#input

For the focusing problem. I think it is not a bug. You should handle it yourself.

**Comment by lochstar** [comment]
Thanks for pointing out the incorrect slot naming.

Yep, the hard-coded tab-index is what I identified as the issue. It would be good to be able to set/unset if required.

**Comment by ziyoung** [comment]
ref #11875

**Comment by dreadful44** [comment]
Hi @ziyoung. You give reference but still this issue exist.
